application/wrapper/storage_mock.go

```diff
@@ -0,0 +1,139 @@
+package wrapper
+
+import (
+	"math/big"
+	"time"
+
+	"github.com/MadBase/MadNet/dynamics"
+	"github.com/dgraph-io/badger/v2"
+)
+
+func MakeMockStorageGetter() *MockStorageGetter {
```

Comment:
Did we investigate mocking out the `Storage` interface and using something like [gomock](https://github.com/golang/mock) to create something that can be extended for testing?